### ISSUE TITLE
Remove or update Sort Table by in Searchbar

### ISSUE THREAD

**Issue by joomlahenk** [joomlahenk/fabrik#24]
The text in the sort selection box needs to be updated. However, the sort selection box is not so usefull as we can already use the sort in the column header. So the sort selection box can be removed. Joomla has no simple option (yet) to do this.

**Comment by trob** [comment]
It's in administrator\components\com_fabrik\forms\filter_elements.xml  etc.

**Comment by joomlahenk** [comment]
Yes, I know (put it there myself), but I looking for a way to remove the sort selector. You can already sort columns using the column headers

**Comment by trob** [comment]
You can remove the  "<field ... fullordering...." in these xml files.

**Comment by joomlahenk** [comment]
Working and merging
